Python/Product/Common/Strings.resx

```diff
@@ -2876,16 +2876,22 @@ Please specify at least one package.</value>
   <data name="CondaEnvNotDetected" xml:space="preserve">
     <value>Conda environment not detected</value>
   </data>
-  <data name="ConfigurePytestTextInfoBarIgnoreAction" xml:space="preserve">
+  <data name="PyTestIgnoreInfoBarAction" xml:space="preserve">
     <value>Don't show this message again</value>
   </data>
-  <data name="ConfigurePytestInfoBarEnableAction" xml:space="preserve">
+  <data name="PyTestEnableInfoBarAction" xml:space="preserve">
     <value>Enable pytest</value>
   </data>
-  <data name="ConfigurePytestTxtInfoBarMessage" xml:space="preserve">
-    <value>A pytest configuration file '{0} was detected in '{1}', but pytest is not enabled</value>
+  <data name="PyTestConfigurationFileFound" xml:space="preserve">
+    <value>A pytest configuration file was detected in "{0}", but pytest is not enabled</value>
   </data>
-  <data name="ConfigurePytestInfoBarInstallAndEnableAction" xml:space="preserve">
-    <value>Install and enable PyTest</value>
+  <data name="PyTestInstallAndEnableInfoBarAction" xml:space="preserve">
+    <value>Install and enable pytest</value>
+  </data>
+  <data name="PyTestNotInstalled" xml:space="preserve">
+    <value>Pytest in enabled in "{0}", but it is not installed in the active virtual environment</value>
```

Comment:
is enabled (not in enabled)
active environment (remove virtual)
use single quotes, not double quotes, for consistency with others

Reply:
dot at the end of sentence